src/tools/sssctl/sssctl_config.c

```diff
@@ -63,7 +63,10 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
 
     /* Open config file */
     ret = sss_ini_config_file_open(init_data, SSSD_CONFIG_FILE);
-    if (ret != EOK) {
+    if (ret == ENOENT) {
+        ERROR("File "SSSD_CONFIG_FILE" does not exist.\n");
```

Comment:
Sorry for nitpicking but it is not very user-friendly for translators.
On other places we use `"%1$s"`. Check src/tools/sss_useradd.c